### ISSUE TITLE
Propcache is not reproducible

### ISSUE THREAD

**Issue by bbhtt** [aio-libs/propcache#68]
### Please confirm the following

- [X] I agree to follow the [aio-libs Code of Conduct][CoC]

[CoC]: ../../.github/blob/master/CODE_OF_CONDUCT.md

- [X] I have checked the [current issues][issue search] for duplicates.

[issue search]: ../search?q=is%3Aissue&type=issues

- [X] I understand this is open source software provided for free and that I might not receive a timely response.
- [X] I am positive I am **NOT** reporting a (potential) security
vulnerability, to the best of my knowledge. *(These must be shared by
submitting [this report form][vulnerability report form] instead, if
any hesitation exists.)*

[vulnerability report form]: ../security/advisories/new

- [ ] I am willing to submit a pull request with reporoducers as xfailing test cases or even entire fix. *(Assign this issue to me.)*

### Describe the bug

See diffoscope result https://freedesktop-sdk.gitlab.io/-/freedesktop-sdk/-/jobs/8191608665/artifacts/result_folder/components/python3-propcache.bst/index.html

[diffoscope.txt](https://github.com/user-attachments/files/17565643/diffoscope.txt) (change to .HTML)

Seems to be from here https://github.com/aio-libs/propcache/blob/73fc000cc8b2609ce9479207c4d3b84a03a22576/packaging/pep517_backend/_backend.py#L224-L228

### To Reproduce

Build it in freedesktop sdk https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/master/CONTRIBUTING.md?ref_type=heads#testing-locally and compare reproducibility. 

### Expected behavior

It should be reproducible

### Logs/tracebacks

```python-traceback
N/A
```

### Python Version

```console
3.13
```

### propcache Version

```console
v0.2.0
```

### OS

Linux (Freedesktop SDK)

### Additional context

N/A

**Comment by webknjaz** [comment]
Yes, thiw is the same as with yarl et al. I don't remember if I settled on the way to address this yet. Need to dig into the old issues. I think some downstreams just patch it out for now.

**Comment by webknjaz** [comment]
Oh, I think I wanted to implement a PEP517 config setting with the flag..

**Comment by webknjaz** [comment]
> Oh, I think I wanted to implement a PEP517 config setting with the flag..

Found where I mentioned this in the past: https://github.com/aio-libs/yarl/issues/992#issuecomment-1954455381
